To: vim_dev@googlegroups.com
Subject: Patch 7.3.214
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.214
Problem:    The text displayed by ":z-" isn't exactly like old Vi.
Solution:   Add one to the start line number. (ChangZhuo Chen)
Files:      src/ex_cmds.c


*** ../mercurial/vim73/src/ex_cmds.c	2011-05-19 14:30:07.000000000 +0200
--- src/ex_cmds.c	2011-05-19 14:23:33.000000000 +0200
***************
*** 4097,4104 ****
      switch (*kind)
      {
  	case '-':
! 	    start = lnum - bigness * (linenr_T)(x - kind);
! 	    end = start + bigness;
  	    curs = end;
  	    break;
  
--- 4097,4104 ----
      switch (*kind)
      {
  	case '-':
! 	    start = lnum - bigness * (linenr_T)(x - kind) + 1;
! 	    end = start + bigness - 1;
  	    curs = end;
  	    break;
  
*** ../vim-7.3.213/src/version.c	2011-06-12 21:51:01.000000000 +0200
--- src/version.c	2011-06-12 22:02:20.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     214,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
170. You introduce your wife as "my_lady@home.wife" and refer to your
     children as "forked processes."

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
